content/docs-v1/figure-images.md

```diff
@@ -20,117 +18,93 @@ Quire does not require a specific image file format or size, but we have some re
 - Watch out for file sizes, especially on animated gifs which can get to be multiple megabytes quite quickly. Use {{< q-def "Image Optimization" >}} software when possible, and consider the total number of images on a given page when choosing sizes.
 
 {{< q-class "box tip" >}}
-To include deep-zooming images in your Quire project please see the [*Zooming Images with IIIF*](/docs-v1/zooming-images) section of this guide.
+- To include deep-zooming images in your Quire project please see the [*Zooming Images with IIIF*](/docs-v1/zooming-images) section of this guide.
 {{< /q-class >}}
 
 ## Create a figures.yaml File for Figure Image Metadata
 
-For most publications, or, at least, those with more than just a handful of images, figures and all their associated attributes can be listed in the `figures.yaml` file, which should be placed in your `data` folder. This figure image metadata can then be called from wherever you need it in your project with a shortcode. See the API-DOCs section for [complete details on possible figure attributes](/docs-v1/for-developers/#figure_list), but below there is a very simple example with `id` and `src` (required attributes) and `alt` (recommended attribute).
+For most publications, or, at least, those with more than just a handful of images, figures and all their associated attributes can be listed in the `figures.yaml` file, which lives in your `/content/_data` directory. The figure image metadata, along with the image itself, can then be pulled into your project through the use of a shortcode. See the API-DOCs section for [complete details on possible figure attributes](/docs-v1/for-developers/#figure_list). Below is an example of two `figures.yaml` entries with the required attributes: `id` and `src`.
```

Comment:
> For most publications, or, at least, those with more than just a handful of images

While in the Hugo version, users could add image info (label, caption, alt text) within the shortcode and forgo the use of `figures.yaml`, that's not an option any more.